packages/sdk/app-graph/src/graph-builder.test.ts

```diff
@@ -105,76 +118,92 @@ describe('GraphBuilder', () => {
       const name = signal('default');
       builder.addExtension(
         'connector',
-        connector(() => [{ id: 'named', type: 'type', data: name, properties: { label: name.value } }]),
+        defineExtension({
+          connector: () => [{ id: EXAMPLE_ID, type: EXAMPLE_TYPE, data: name, properties: { label: name.value } }],
+        }),
       );
       const graph = builder.build();
 
-      expect(graph.nodes(graph.root)[0]?.properties.label).to.equal('default');
+      const [node] = graph.nodes(graph.root);
+      expect(node.properties.label).to.equal('default');
 
       name.value = 'updated';
-
-      expect(graph.nodes(graph.root)[0]?.properties.label).to.equal('updated');
+      expect(node.properties.label).to.equal('updated');
     });
 
     test('removes', () => {
       const builder = new GraphBuilder();
       const nodes = signal([
-        { id: 'first', type: 'type', data: 1 },
-        { id: 'second', type: 'type', data: 2 },
+        { id: 'first', type: EXAMPLE_TYPE, data: 1 },
```

Comment:
'item-1', 'item-2' i think is clearer (more realistic and moves towards generating these for larger tests).